### PR TITLE
Fix SSL Certs

### PR DESCRIPTION
**Pull request by ryanshand**
This fixes the SSL cert issue.

### CODE REVIEW

File: bin/hisensetv/__main__.py [hunk 1]
```diff
@@ -25,7 +27,7 @@ def main():
         "--get",
         action="append",
         default=[],
-        choices=["sources", "volume", "state"],
```
Comment (chinedu40) [inline bin/hisensetv/__main__.py:28]: Why has the state choice been removed?

Comment (ryanshand) [reply bin/hisensetv/__main__.py]: must have been an issue when I copied from the original. I just did a compare in notepad++ to make sure i didnt miss any others, and have put state back in.

File: bin/hisensetv/__main__.py [hunk 2]
```diff
@@ -88,26 +90,28 @@ def main():
...
     args = parser.parse_args()
-
-    if args.verbose:
-        level = logging.DEBUG
-    else:
-        level = logging.INFO
-
+    
+    startingpoint1 = '/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/Hisensecerts/rcm_certchain_pem.cer'
+    startingpoint2 = '/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/Hisensecerts/rcm_pem_privkey.pkcs8'
```
Comment (MrAsterisco) [inline bin/hisensetv/__main__.py:95]: I don't think this is a good solution. I understand that we need to pass the full path for this to work, but there's gotta be a better way. Maybe it is possible to get the `node_modules` directory in some way?

File: bin/hisensetv/__main__.py [hunk 3]
```diff
@@ -88,26 +90,28 @@ def main():
...
+    
+    level = logging.INFO
+    	
     root_logger = logging.getLogger()
     stream_handler = logging.StreamHandler()
     formatter = logging.Formatter(
-        fmt="[{asctime}] [{levelname:<8}] {message}", style="{"
-    )
+            fmt="[{asctime}] [{levelname:<8}] {message}", style="{"
+        )
```
Comment (MrAsterisco) [inline bin/hisensetv/__main__.py:104]: You might want to check the indentation here.

File: bin/hisensetv/__main__.py [hunk 4]
```diff
@@ -88,26 +90,28 @@ def main():
...
     root_logger.setLevel(level)
     logger = logging.getLogger(__name__)
 
     if args.no_ssl:
-        ssl_context = None
+     ssl_context = None
     else:
-        ssl_context = ssl._create_unverified_context()
+       ssl_context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+       ssl_context.load_cert_chain(certfile=startingpoint1, keyfile=startingpoint2)
```
Comment (MrAsterisco) [inline bin/hisensetv/__main__.py:114]: Same issue with the indentation here as well.

**Review by MrAsterisco — COMMENTED** [review]
Thanks for this PR, it looks promising. I'm not entirely convinced about referencing a static path to the SSL keys, it just looks too error-prone and not very future-proof. I am also okay in merging this, if you feel like there's no other solution.

**Review by MrAsterisco — APPROVED** [review]

**Comment by chinedu40** [comment]
I have made some additional changes to this on another branch of @ryanshand fork. I was able to confirm that his code works to set a path for the certificates and i added some if.args statements so that we can preserve @MrAsterisco orginal code while providing a way to pass --certfile and --keyfile argument to load the certificates if people need to. Hope this helps

**Comment by ryanshand** [comment]
this one can be cancelled

**Pull request closed without merging**